packages/react-core/src/components/WizardComposable/examples/WizardKitchenSink.tsx

```diff
@@ -0,0 +1,155 @@
+              <DrawerActions>
+                <DrawerCloseButton onClick={() => setIsDrawerExpanded(false)} />
+              </DrawerActions>
+            </DrawerHead>
+          </DrawerPanelContent>
+        }
+      >
+        <WizardComposableBody>
+          <Flex direction={{ default: 'column' }}>
+            {!isDrawerExpanded && (
+              <Button isInline variant="link" onClick={() => setIsDrawerExpanded(prevExpanded => !prevExpanded)}>
+                Open drawer
+              </Button>
+            )}
+
+            <FormGroup fieldId="some-text-input" label="Text input label">
+              <TextInput aria-label="Some Text Input Label" />
+            </FormGroup>
+          </Flex>
+        </WizardComposableBody>
+      </DrawerContent>
+    </Drawer>
+  );
+};
+
+const StepWithCustomFooter = () => {
+  const { onNext: goToNextStep, onBack, onClose } = useWizardContext();
+  const [isLoading, setIsLoading] = React.useState(false);
+
+  async function onNext(goToStep: () => void) {
+    setIsLoading(true);
+    await new Promise(resolve => setTimeout(resolve, 2000));
+    setIsLoading(false);
+
+    goToStep();
+  }
+
+  const footer = React.useMemo(
+    () => (
+      <div className={css(styles.wizardFooter)}>
+        <Button
+          variant="primary"
+          type="submit"
+          onClick={() => onNext(goToNextStep)}
+          isLoading={isLoading}
+          isDisabled={isLoading}
+        >
+          Async Next
+        </Button>
+        <Button variant="secondary" onClick={onBack}>
+          Back
+        </Button>
+        <Button variant="link" onClick={onClose}>
+          Cancel
+        </Button>
+      </div>
+    ),
+    [isLoading, onBack, onClose, goToNextStep]
+  );
+  useWizardFooter(footer);
+
+  return <>Step 3 content w/ custom async footer</>;
+};
+
+export const WizardKitchenSink: React.FunctionComponent = () => (
+  <WizardComposable
+    height={400}
+    header={<WizardHeader title="You're a wizard, Harry" closeButtonAriaLabel="Close header" />}
+    footer={<CustomWizardFooter />}
+    nav={{ forceStepVisit: true, isExpandable: true }}
+  >
+    <WizardComposableStep id="first-step" name="First step" includeStepBody={false}>
+      <StepContentWithDrawer />
+    </WizardComposableStep>
+    <WizardComposableStep
+      id="second-step"
+      name="Second step"
+      steps={[
+        <WizardComposableStep id="sub-step-one" key="substep-one" name="Substep 1">
+          Substep 1 content
+        </WizardComposableStep>,
+        <WizardComposableStep id="sub-step-two" key="substep-two" name="Substep 2">
+          Substep 2 content
+        </WizardComposableStep>
+      ]}
+    />
+    <WizardComposableStep id="third-step" name="Third step" navItem={<CustomNavItem />}>
+      <StepWithCustomFooter />
+    </WizardComposableStep>
+    <WizardComposableStep id="fourth-step" name="Fourth step">
+      Step 4 content
```

Comment:
Could the 4th step include a drawer maybe?

Reply:
I think since step 1 has a drawer already, it's probably not needed. What I can do though is make it a custom component (not use WizardComposableStep), which would be another form of uniqueness to add to the sink.

Reply:
Oh I didn't even realize it for step 1.
So if you want to add more uniqueness to the sink, that's up to you.